manifests/0000_50_olm_07-olm-operator.deployment.yaml

```diff
@@ -48,6 +55,18 @@ spec:
                 fieldPath: metadata.namespace
           - name: OPERATOR_NAME
             value: olm-operator
+          
+          volumeMounts:
+          - mountPath: /var/run/secrets/serving-cert
+            name: serving-cert
+          
```

Comment:
nit: I think we want to clean up the white-space chomping in the template in a future PR.